internal/validitywindow/handler_test.go

```diff
@@ -18,20 +17,21 @@ import (
 
 func TestBlockFetcherHandler_FetchBlocks(t *testing.T) {
 	tests := []struct {
-		name         string
-		setupBlocks  func() map[uint64]ExecutionBlock[container]
-		blockHeight  uint64
-		minTimestamp int64
-		wantErr      bool
-		wantBlocks   int
+		name          string
+		setupBlocks   func() map[uint64]ExecutionBlock[container]
```

Comment:
idea for separate / follow up PR - all the parameters are only controlling an interface with a single function, can we just define that function as needed?